server.js

```diff
@@ -20,7 +37,58 @@ app.use(bodyParser.json())
 
 // Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Happy thoughts API')
+})
+
+app.get('/thoughts', async (req, res) => {
+  try {
+    res.status(200).json(await Thought.find().sort({ createdAt: 'desc' }).limit(20).exec())
+  } catch (error) {
+    res.status(400).json({ message: 'Could not get thoughts', error })
+  }
+})
+
+app.post('/thoughts', async (req, res) => {
+  try {
+    res.status(201).json(await new Thought({ message: req.body.message }).save())
+  } catch (error) {
+    res.status(400).json({ message: 'Could not save thought', error: error.errors.message.message })
+  }
+})
+
+// delete not yet used in frontend
+app.delete('/thoughts/:id', async (req, res) => {
```

Comment:
Even if you do not use them in the front end yet, it is nice to see that you have prepared. 👍👌